util/filter_bench.cc

```diff
@@ -287,6 +290,15 @@ static uint32_t DryRunHash64(Slice &s) {
   return Lower32of64(GetSliceHash64(s));
 }
 
+const std::shared_ptr<const FilterPolicy> &GetPolicy() {
+  static std::shared_ptr<const FilterPolicy> policy;
+  if (!policy) {
+    policy = BloomLikeFilterPolicy::Create(
+        BloomLikeFilterPolicy::kAllFixedImpls[FLAGS_impl], FLAGS_bits_per_key);
```

Comment:
Should you check that FLAGS_impl is in range?

Reply:
Done using `at()`